### PR TITLE
remove workflows

### PR DESCRIPTION
**Pull request by thirdeyenick**
there are no workflows needed anymore, so we can remove them

### CODE REVIEW

File: .github/workflows/redirect.yml [hunk 1]
```diff
@@ -1,49 +0,0 @@
...
-        with:
-          name: redocusaurus
-          path: public/
-      - name: Setup Pages
-        uses: actions/configure-pages@v3
-      - name: Upload artifact
-        uses: actions/upload-pages-artifact@v1
-        with:
-          path: public/
-      - name: Deploy to GitHub Pages
```
Comment (ctrox) [inline .github/workflows/redirect.yml:47]: Are you sure we don't need deploy to pages?

Comment (ctrox) [reply .github/workflows/redirect.yml:47]: Ah ok as we just have the static `index.html` this seems to work. But the links in the html look a bit off to me:

```bash
$ curl https://docs.nineapis.ch
<!DOCTYPE html>
<html>
  <head>
    <title>Nine API docs</title>
    <link rel="icon" type="image/x-icon" href="favicon.ico">
    <meta http-equiv="Refresh" content="0; url=//docs.nine.ch/api" />
  </head>
  <body>
    <p>API docs have moved to <a href="//docs.nine.ch/api">docs.nine.ch</a>.</p>
  </body>
</html>
```

Comment (thirdeyenick) [reply .github/workflows/redirect.yml:47]: You mean the "//docs.nine.ch/api" ref? That will just use whatever protocol was used when hitting docs.nineapis.ch. But you are right. We should always use https.

Comment (ctrox) [reply .github/workflows/redirect.yml:47]: Ah, did not know that would work. As we have a redirect to https anyways it should not matter.

**Pull request merged**